.github/workflows/ci-test.yml

```
@@ -37,11 +54,29 @@ jobs:
+      # Wait mysql ready
+      - name: Wait for MySQL to be ready
+        env:
+          MYSQL_ROOT_PASSWORD: ${{ secrets.MYSQL_PASSWORD }}
+        run: |
+          until mysqladmin ping -h"127.0.0.1" -u root -p"${MYSQL_ROOT_PASSWORD}" --silent; do
+            echo "Waiting for database connection..."
+            sleep 2
+          done
+
+      # Run MySQL schema script
+      - name: Run MySQL Schema
```

actionlint

step must run script with "run" section or run action with "uses" section

(syntax-check)